[DRAFT] Lifecycle graphic tryout

egpbos opened the pull request:
During the Editorial Board meeting, we discussed making a nice cycle graphic like the RDMkit has to visualize and link to advise pertaining to different parts of the research software life cycle. This branch is meant as a test ground for me and @sparkslabs (and others who are interested!) to see if we can easily adopt and adapt the RDMkit code for this.

For now I only copy-pasted the cycle that they have on their [data life cycle page](https://rdmkit.elixir-europe.org/data_life_cycle). They also have a smaller version in their side bar, which was added in [this PR](https://github.com/elixir-europe/rdmkit/commit/999ea33004dd8d75a8b76753c8e8302b9ab4a620). The sidebar one will be a bit more involved, because it needs also a modified sidebar.html page, which is a bit annoying to maintain... although perhaps this has since been included in ETT in some way, which this line hints at: https://github.com/elixir-europe/rdmkit/blob/5d2dfa67a1b4d7c44db800efe1633d8d2d4e0e43/_data/sidebars/data_management.yml#L8

Let's figure it out!

egpbos commented:
So, as you see in the code, the icons in the wheel are actually hardcoded SVG or similar drawings :) Not sure how we'll replace that, perhaps there is some repository with icons like this in SVG format?

anenadic commented:
> So, as you see in the code, the icons in the wheel are actually hardcoded SVG or similar drawings :) Not sure how we'll replace that, perhaps there is some repository with icons like this in SVG format?

Indeed - I think I asked Bert from ETT a while back abotu the diagram and these are custom made SVG snippets that are clickable. I think perhaps we should investigate just using some Javascripp drawing library that draws flowchart-like (or similar) diagrams from e.g. a JSON file that can be updated and tweaked.

sparkslabs commented:
Just a comment to say that I've taken a look at this, and I'm not sure that this is necessarily how I view the lifecycle. I've been letting myself mull it over rather than comment here. I think this is a reflection of the different types of research software that exist. (Software that supports research vs software that is itself exploratory research)

In the process of mulling that, I've also noted that the three tiers are listed in a different order on that page from the three tiers page, so I'm going to add an issue for that and put in a draft PR for that instead! After that I'll pull my ideas together more cogently!

(On that note, shouldn't this be a draft pr rather than a PR - unless it's ready for merge?)

egpbos commented:
Indeed, this is a draft PR, not ready to merge, but in order to trigger Netlify builds I set it to normal and put the big [DRAFT] tag in the title :)

egpbos commented:
I rebased this branch on main and added a new draft. I "vibe coded" this, as the kids call it these days ;)

You can find it in a separate HTML file this time: https://deploy-preview-312--everse-rsqkit-testing.netlify.app/pages/cycle.html

What do you think, is this helpful to guide people towards the content they need?

sparkslabs commented:
I think this is superceded by https://github.com/EVERSE-ResearchSoftware/RSQKit/issues/386 ? 
Do you agree @egpbos ?
